app/views/_layouts/content-simple.nunjucks

```diff
@@ -9,13 +9,29 @@
 
     {% block main_content_before %}
       {% for component in before %}
-        {{ components[component.type](component.value) }}
+        {% if component.type == 'markdown' %}
+          <div class="page-section">
+            <div class="reading-width">
+              {{ components[component.type](component.value) }}
+            </div>
+          </div>
+        {% else %}
+          {{ components[component.type](component.value) }}
```

Comment:
Another option is to pass the `region` to the macro and have it check `if region == ... then wrap in those divs`.

This way the macro *knows* what the region is and can decide how to display itself based on that.

What do you think @tyom ?